recipes/execnet/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "execnet" %}
+{% set version = "1.4.1" %}
+{% set hash_type = "sha256" %}
+{% set hash = "f66dd4a7519725a1b7e14ad9ae7d3df8e09b2da88062386e08e941cafc0ef3e6" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - setuptools_scm
+  run:
+    - python
+    - apipkg >=1.4
```

Comment:
There will be a race condition and you'll need to restart the CIs when `apipkg` is ready.